helm/alfresco-infrastructure/values.yaml

```diff
@@ -18,6 +25,12 @@ nginx-ingress:
       enabled: true  
     config:
       ssl-redirect: "false"
-
-alfresco-identity-service:
-  enabled: true
\ No newline at end of file
+      server-tokens: "false"
+ 
+alfresco-infrastructure:
+  alfresco-identity-service:
```

Comment:
Was this not working when under root?  Won't this change require other charts consuming infrastructure to update their values with an additional `alfresco-infrastructure`?

Reply:
The conditions did not change @rgauss , i just added these here for reference.